modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/PolicyDetails/policy-navigator-details/driver-info/index.tsx

```diff
@@ -0,0 +1,146 @@
+	dataJSON: dataJSONType;
+	email: string;
+	phone: string;
+};
+
+const DriverInfo = ({dataJSON, email, phone}: ApplicationPolicyDetailsType) => {
+	const applicationData = dataJSON;
+
+	return (
+		<div>
+			{applicationData?.driverInfo?.form.map(
+				(curentDriver: PolicyDetailsType, indexDriver: number) => (
+					<div
+						className="bg-neutral-0 pl-6 policy-detail-border pr-6 pt-6"
+						key={indexDriver}
+					>
+						<div className="d-flex flex-row flex-wrap justify-content-between">
+							{indexDriver !== 0 && (
+								<div className="align-self-start col-12 layout-line mb-6 mt-1"></div>
+							)}
+
+							<div className="align-self-start">
+								<h5>
+									{`${
+										curentDriver?.firstName ?? 'No data'
+									}, ${
+										applicationData?.contactInfo
+											?.dateOfBirth
+											? calculatedAge(
+													applicationData?.contactInfo
+														?.dateOfBirth
+											  )
+											: 'No data'
+									}`}
+								</h5>
+							</div>
+
+							<div className="align-self-start">
+								<p className="mb-1 text-neutral-7 w-100">DOB</p>
+
+								<div className="mb-3">
+									{applicationData?.contactInfo?.dateOfBirth
+										? applicationData?.contactInfo
+												?.dateOfBirth
+										: 'No data'}
+								</div>
+
+								<p className="mb-1 text-neutral-7 w-100">
+									Education
+								</p>
+
+								<div className="mb-3">
+									{curentDriver?.highestEducation
+										? curentDriver?.highestEducation
+										: 'No data'}
+								</div>
+
+								<p className="mb-1 text-neutral-7 w-100">
+									Email
+								</p>
+
+								<a className="mb-3 text-break" href={email}>
+									{email ? email : 'No data'}
+								</a>
+							</div>
+
+							<div className="align-self-start">
+								<p className="mb-1 text-neutral-7 w-100">
+									Gender
+								</p>
+
+								<div className="mb-3">
+									{curentDriver?.gender
+										? curentDriver?.gender
+										: 'No data'}
+								</div>
+
+								<p className="mb-1 text-neutral-7 w-100">
+									Occupation
+								</p>
+
+								<div className="mb-3">
+									{curentDriver?.occupation
+										? curentDriver?.occupation
+										: 'No data'}
+								</div>
+
+								<p className="mb-1 text-neutral-7 w-100">
+									Phone
+								</p>
+
+								<div className="mb-3">
+									{phone ? phone : 'No data'}
+								</div>
+							</div>
+
+							<div className="align-self-start">
+								<p className="mb-1 text-neutral-7 w-100">
+									Marital Status
+								</p>
+
+								<div className="mb-3">
+									{curentDriver?.maritalStatus
+										? curentDriver?.maritalStatus
+										: 'No data'}
+								</div>
+
+								<p className="mb-1 text-neutral-7 w-100">
+									Credit rating
+								</p>
+
+								<div className="mb-3">
+									{curentDriver?.creditRating
+										? curentDriver?.creditRating
+										: 'No data'}
```

Comment:
Replace ternary to "??" as the other suggestions.

Also, this entire code is very repetitive, it's possible to simplify?

Something like:

```js
function SomeInterestingName({label, value}) {
 return (
  <div>
	<p className="mb-1 text-neutral-7 w-100">
 		{label}
 	</p>

 	<div className="mb-3">
 	  {value ?? 'No data'}
         </div>
   </div>
  )
}

const values = [
{
   label: "Phone",
   value: phone
},
{
   label: "Marital Status",
   value: currentDriver?.maritalStatus
}, ...
]

values.map((value, index) => (
    <SomeInterestingName key={index} {...value} />
)

```

Find a better name to the function and the values array, that way you could save a lot of line of code